app/src/main/java/espertolabs/esperto_ble_watch/SummaryActivity.java

```diff
@@ -681,4 +695,13 @@ else if(obj == userSteps){
             Log.i("State", "Step count state has changed");
         }
     }
+
+    public void signOut(){
+        //TODO: MOVE ALL COGNITO STUFF TO A HANDLER
+        CognitoUserPool userPool = new CognitoUserPool(getApplicationContext(), getString(R.string.cognito_userpool_id), getString(R.string.cognito_client_id), getString(R.string.cognito_client_secret), Regions.fromName(getString(R.string.cognito_region)));
```

Comment:
Not required, but maybe have each parameter on its own line for readability.

Reply:
gotcha